core/local/watcher.js

```diff
@@ -20,20 +19,27 @@ export interface Watcher {
 }
 */
 
-const log = logger({
-  component: 'LocalWatcher'
-})
-
 function build (syncPath /*: string */, prep /*: Prep */, pouch /*: Pouch */, events /*: EventEmitter */, ignore /*: Ignore */) /*: Watcher */ {
+  let watcher = 'atom'
+  if (process.platform === 'darwin') {
+    watcher = 'chokidar'
+  }
   const env = process.env.COZY_FS_WATCHER
```

Comment:
Wouldn't using the same name as the environment variable be easier to understand? E.g.:

```js
const { COZY_FS_WATCHER } = process.env
```